components/toggleButton/toggleButton.js

```diff
@@ -1,85 +1,49 @@
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import React from 'react';
 import { getClassNamesWithMods } from '../_helpers';
 
-/**
- * ToggleButton component.
- */
-class ToggleButton extends Component {
-  /**
-   * Initialize default state
-   */
-  state = {
-    activeItem: 0,
-  }
-
-  /**
-   * Handlers onClick event
-   *
-   * @method handleSelectItem
-   * @param {Object} event
-   * @private
-   */
-  handleSelectItem = (event) => {
-    event.stopPropagation();
+export default function ToggleButton(props) {
+  const {
+    className,
+    mods = [],
+    handleSelect,
+    items,
+    selectedIndex,
+  } = props;
 
-    const index = this.props.items.indexOf(event.target.textContent);
-    this.setState({ activeItem: index });
-    (typeof this.props.handleSelect === 'function') && this.props.handleSelect(index);
+  if (!Array.isArray(items) || items.length < 2) {
+    return <noscript />;
```

Comment:
What react version should we support? If it's 15 then we can start using `null`.

Reply:
Good point. However, the purpose of this PR it's to convert the component to stateless. That was based on the master which still uses R14 on the package.json

Nevertheless, if someone is taking care of that,  I can wait, rebase and change the code.

Reply:
Ok I took care of it 🙄  and will change that.